packages/style-utilities/src/styles/getFocusStyle.ts

```diff
@@ -92,6 +93,7 @@ function _getFocusStyleInternal(theme: ITheme, options: IGetFocusStylesOptions =
       [`.${IsFocusVisibleClassName} &${isFocusedOnly ? ':focus' : ''}:after`]: {
         content: '""',
         position: 'absolute',
+        pointerEvents,
```

Comment:
I originally set `pointerEvents: none` here, similar to `getInputFocusStyle()` function in the same file.
Even though that should be intended and not break anything, it affects a lot of components in the whole library.
For that reason I decided to go the safer path making the `pointerEvents` configurable and set them to `none` only in the `SplitButton`.